website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "IDE glossary"
+id: ide-glossary
+description: "Develop, test, run, and build in the Cloud IDE. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly"
+sidebar_label: IDE glossary
+tags: [IDE]
+---
+
+The [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) is an interface tool for developers to effortlessly build, test, run, and version-control their dbt projects, and enhance data governance — all from the convenience of your browser. Use the Cloud IDE to compile dbt code into SQL and run it against your database directly -- no command line required!
+
+This page offers comprehensive definitions and terminology of user interface elements, allowing you to navigate the IDE landscape with ease.
+
+## Basic layout
+
+The IDE is designed to streamline your workflow, and features a popular user interface and layout with all files and folders on the left, editor on the right, and command and console information at the bottom. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-basic-layout.jpg" width="100%" title="The Cloud IDE layout includes version control on the upper left, files/folders on the left, editor on the right and command/console at the bottom"/>
+
+The basic layout of the IDE landing page with the following areas:
+
+1. **Git repository link &mdash;** Clicking the Git repository link, located on the upper left of the IDE, takes you to your repository on the same active branch. 
```

Comment:
Consider using bullets because steps indicate we're conveying an action the user needs to take. UNLESS you have corresponding numbers on the diagram like you do later.

Reply:
I think the images should come before the explanations. Like you have in "Editing Features" and "Console" sections

Reply:
gotcha! i did have corresponding numbers on the diagram but i removed it because it became too much in terms of the UX (cross referencing the image and numbers) - but forgot to change to bullets!